examples/set.yaml

```diff
@@ -8,7 +8,7 @@ schedule:
     one:
       with:
         type: io.serverlessworkflow.samples.events.trigger.v1
-do: 
+do:
```

Comment:
Same comment as before: it is still a runtime expression instead of an object, like initially.